src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -176,6 +170,12 @@ It is the responsibility of the Asset Integrator to settle funds with a merchant
 
 An idempotency key is used for `HTTP POST` requests. Centrapay **does not** guarantee endpoints will be called only once per transaction. It is expected that you will enforce transaction idempotency. In the event that the idempotency is violated, Centrapay expects a `200 OK` response as described in the endpoint specification.
 
+### Errors
+
+A 2XX response should be returned from all endpoints for both successful and failed transaction attempts unless an unexpected error has occurred in your system. Centrapay will retry a transaction attempt when an HTTP response status code ≥ 500 is thrown.
+
+Centrapay will also use the [Get Transaction Endpoint](#get-transaction-endpoint) to resolve transactions if errors are thrown.
```

Comment:
@AndyClifford Suggestion: Similar to my other comment - avoid tacking things on with `also`

Change: I think this section should describe how to use the `failed` status + `failureReason` too